contracts/OwnerUpgradeableProxy_sol_OwnerUpgradeableProxy.abi

```diff
@@ -0,0 +1 @@
+[{"constant":false,"inputs":[],"name":"resume","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":false,"inputs":[{"name":"target","type":"address"}],"name":"upgrade","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"paused","outputs":[{"name":"val","type":"bool"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":false,"inputs":[],"name":"pause","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"owner","outputs":[{"name":"addr","type":"address"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":true,"inputs":[],"name":"target","outputs":[{"name":"addr","type":"address"}],"payable":false,"stateMutability":"view","type":"function"},{"inputs":[],"payable":false,"stateMutability":"nonpayable","type":"constructor"},{"payable":true,"stateMutability":"payable","type":"fallback"},{"anonymous":false,"inputs":[{"indexed":true,"name":"target","type":"address"}],"name":"Upgraded","type":"event"},{"anonymous":false,"inputs":[],"name":"Paused","type":"event"},{"anonymous":false,"inputs":[],"name":"Resumed","type":"event"}]
```

Comment:
nit: pretty print these ABIs

Reply:
Maybe in a follow up, we can format these, and associate the `.abi` extension with JSON highlighting: https://github.com/gochain-io/web3/blob/master/.gitattributes

Reply:
Sorry about that. Those `.abi` files weren't supposed to be included. They're formatted in the `assets` folder. I also moved the `sol` files over to `assets` so they're colocated.